src/libPMacc/include/algorithms/reverseBits.hpp

```diff
@@ -48,7 +48,7 @@ reverseBits(T value)
     /* init with value (to get LSB) */
     T result = value;
     /* extra shift needed at end */
-    int s = sizeof(T) * CHAR_BIT - 1;
+    int s = std::numeric_limits<T>::digits - 1;
```

Comment:
please do not change this algorithm together with the rest of this pull request.

`std::numeric_limits<>` is not defined for 64bit types in [C++98](http://www.cplusplus.com/reference/limits/numeric_limits/). Please keep the sold method.